sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsServiceSerializer.cs

```diff
@@ -449,7 +449,7 @@ private static SentenceSentiment ReadSentenceSentiment(JsonElement documentEleme
 
             if (documentElement.TryGetProperty("sentiment", out JsonElement sentimentValue))
             {
-                sentiment = (SentimentLabel)Enum.Parse(typeof(SentimentLabel), sentimentValue.ToString(), ignoreCase: true);
+                sentiment = (SentenceSentimentLabel)Enum.Parse(typeof(SentenceSentimentLabel), sentimentValue.ToString(), ignoreCase: true);
```

Comment:
If I'm interpreting things correctly, we don't expect the service to return `Mixed` for a sentence, but it may for a document.   Since it appears we trust the service to to the right thing here, it would seem we could consider one unified `Sentiment` enum that is a superset of  all of them.

Is breaking the sentiment into dedicated enumerations intended to help guide customers consuming the return value or prevent them from making a mistake when pushing something to the service?

Reply:
>Is breaking the sentiment into dedicated enumerations intended to help guide customers consuming the return value

this.

>it would seem we could consider one unified Sentiment enum that is a superset of all of them.

Will try this

Reply:
I'm not sure that I'm suggesting it, honestly.    It's likely that I'm overlooking something.   I was trying to understand the context a bit better and, potentially, reduce the number of types that a customer would have to track.   There may be very good reasons to split them.   You understand the service and the usage scenarios far better than I.   :smile:

My point being: if my thought makes sense to you, feel free to run with it but don't be shy about telling me that I'm wrong.

Reply:
Leaving it as is. Thank you Jesse!